homeassistant/components/tessie/strings.json

```diff
@@ -315,5 +315,22 @@
         "name": "[%key:component::update::title%]"
       }
     }
+  },
+  "exceptions": {
+    "not_supported": {
+      "message": "{name} is not supported."
+    },
+    "cable_connected": {
+      "message": "Charge cable is connected."
+    },
+    "already_active": {
+      "message": "{name} is already active."
+    },
+    "already_inactive": {
+      "message": "{name} is already inactive."
+    },
+    "incorrect_pin": {
+      "message": "Incorrect pin."
```

Comment:
Does it make sense to include the name here as well?

Reply:
`Incorrect pin for {name}.`